src/components/ArticleLoader.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ContentLoader from "react-content-loader";
+
+const ArticleLoader = props => (
+  <ContentLoader
+    rtl
+    height={360}
+    width={300}
+    speed={4}
+    primaryColor="#f3f3f3"
+    secondaryColor="#ecebeb"
+    {...props}
+  >
+    <rect x="70" y="15" rx="4" ry="4" width="117" height="6.4" />
```

Comment:
I really like this loader feature you implemented. Nice initiative

Reply:
Thanks 💯  @emekafredy